### PR TITLE
Add progress bar support

### PR DESCRIPTION
**Pull request by rpls**
The accompanying changes to mupq/mupq#124

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -9,4 +9,5 @@
     platform, settings = get_platform(args)
     with platform:
         test = mupq.SimpleTest(settings, platform)
-        test.test_all(rest)
+        if test.test_all(rest):
+            sys.exit(1)
```
Comment (mkannwischer) [inline test.py:13]: Need to import sys here

File: testvectors.py [hunk 2]
```diff
@@ -7,4 +7,5 @@
     platform, settings = get_platform(args)
     with platform:
         test = mupq.TestVectors(settings, platform)
-        test.test_all(rest)
+        if test.test_all(rest):
+            sys.exit(1)
```
Comment (mkannwischer) [inline testvectors.py:11]: Need to import sys here

**Comment by mkannwischer** [comment]
Thank you @rpls! This is great!

I merged https://github.com/mupq/mupq/pull/124.
If you include that commit and fix the the two imports above, we can merge it.

**Comment by rpls** [comment]
Alright, ready to merge I guess?

**Comment by mkannwischer** [comment]
Yes, thanks @rpls!

**Pull request merged**